### PR TITLE
Release Notes for R18

### PR DESCRIPTION
**Pull request by joebochill**
#### Changes proposed in this Pull Request:
- Add developer side release notes
- Add summary

### CODE REVIEW

**Comment by joebochill** [comment]
Need to update the link for the design patterns community file once we have one

**Comment by joebochill** [comment]
Looks like icons8 is having some server issues regarding the link checker...might need to wait on that to settle down, or possibly update our links to the homepage instead since they seem to have restructured their sitemap a bit.

File: src/docs/release-notes/R18/R18.mdx [hunk 1]
```diff
@@ -1,10 +1,36 @@
...
+
+-   Reusable [Authentication Workflow](https://www.npmjs.com/package/@pxblue/angular-auth-workflow) for Angular.
+
 ### Stickersheet Components
 
--   Rewired all the components using Figma's newly-released variants and auto-layout
+-   Re-built all the components using the new Figma variants and auto-layout
 -   Added images, tiles and patterns
 -   Added more components (badges, dialogs, form controls, icon buttons, menus, side sheets, steppers and toggle buttons)
 -   Reorganized the stickersheet structure
```
Comment (huayunh) [inline src/docs/release-notes/R18/R18.mdx:31]: I am too lazy to start my new PR 😛 . Want to insert this for me?

```suggestion
-   Reorganized the stickersheet structure

### Figma Design Patterns

-   Released [a new stickersheet dedicated to our design patterns](https://www.figma.com/community/file/926189711301522231). Added design pattern examples for [forms](https://pxblue.github.io/patterns/forms), [side sheet](https://pxblue.github.io/patterns/overlay#sidebar) and [lists](https://pxblue.github.io/patterns/lists).
```

**Comment by joebochill** [comment]
I switched over the links to use relative links instead of absolute ones (so the app doesn't reload when you click on them). A side effect seems to be that the link with the anchor tag hash didn't seem to work, locally at least. It went to the correct page, but didn't scroll to the heading. If i copied the link and pasted it into a new tab, it worked as expected. Not sure why that would be, so for now I just linked to the top level page instead so it's not confusing.

File: src/docs/release-notes/R18/Summary.mdx [hunk 2]
```diff
@@ -1 +1 @@
-TBD
+Major overhaul of the designer Sticker Sheet, new Reusable [Authentication Workflow](https://www.npmjs.com/package/@pxblue/angular-auth-workflow) for Angular, CLI templates for React projects, and documentation site improvements.
```
Comment (huayunh) [inline src/docs/release-notes/R18/Summary.mdx:1]:
```suggestion
Major overhaul of the Figma Sticker Sheet, new Reusable [Authentication Workflow](https://www.npmjs.com/package/@pxblue/angular-auth-workflow) for Angular, [CLI templates](https://github.com/pxblue/react-cli-templates/tree/master) for React projects, and documentation site improvements.
```

File: src/docs/release-notes/R18/R18.mdx [hunk 3]
```diff
@@ -1,10 +1,36 @@
...
+-   Added a [Mobile Stepper](https://pxblue-components.github.io/angular/?path=/info/components-mobile-stepper--readme) component for Angular
+-   Angular components are now compatible with Angular 10+
+-   Updated React Native components to be compatible with react-native-paper version 4
+
+### Design Patterns
+
+-   Updated content for [Navigation](/patterns/navigation) and [Visualizations](/patterns/visualizations)
+-   Released a new [Design Pattern Stickersheet](https://www.figma.com/community/file/926189711301522231) that will include many examples of all of our [Design Patterns](/patterns), starting with [forms](/patterns/forms), [side sheet](/patterns/overlay) and [lists](/patterns/lists).
+
+### Documentation Site≈
```
Comment (huayunh) [inline src/docs/release-notes/R18/R18.mdx:12]:
```suggestion
### Documentation Site
```

File: src/docs/release-notes/R18/R18.mdx [hunk 4]
```diff
@@ -1,10 +1,36 @@
...
+
+### PX Blue CLI
+
+-   You can now use [templates](https://github.com/pxblue/react-cli-templates) to kick-start your react projects with navigation and authentication
+
+### Reusable Workflows
+
+-   Reusable [Authentication Workflow](https://www.npmjs.com/package/@pxblue/angular-auth-workflow) for Angular.
+
 ### Stickersheet Components
```
Comment (huayunh) [inline src/docs/release-notes/R18/R18.mdx:26]: We outa find some more consistent names for those Figma files

DSM: (Figma) Component Stickersheet
The new design pattern file: (Figma) Design Pattern Stickersheet

File: src/docs/release-notes/R18/R18.mdx [hunk 5]
```diff
@@ -1,10 +1,36 @@
...
+
+### Documentation Site≈
+
+-   Updated [Iconography](/style/iconography) page to enable users to download icons and search by keywords/tags
+-   Added a new page dedicated to helping you build a [Project Identity](/design/project-identity)
+-   Added new holiday themes
+
+### PX Blue CLI
+
+-   You can now use [templates](https://github.com/pxblue/react-cli-templates) to kick-start your react projects with navigation and authentication
```
Comment (huayunh) [inline src/docs/release-notes/R18/R18.mdx:20]:
```suggestion
-   You can now use [templates](https://github.com/pxblue/react-cli-templates) to kick-start your React projects with navigation and authentication
```

File: src/docs/release-notes/R18/R18.mdx [hunk 6]
```diff
@@ -1,10 +1,36 @@
+### Component Libraries
+
+-   Added a [Mobile Stepper](https://pxblue-components.github.io/angular/?path=/info/components-mobile-stepper--readme) component for Angular
+-   Angular components are now compatible with Angular 10+
+-   Updated React Native components to be compatible with react-native-paper version 4
```
Comment (huayunh) [inline src/docs/release-notes/R18/R18.mdx:5]: either 
```suggestion
-   Updated React Native components to be compatible with `react-native-paper` version 4
```

or 
```suggestion
-   Updated React Native components to be compatible with React Native Paper version 4
```

**Review by huayunh — COMMENTED** [review]
Do we want to mention 
* PXBLUE-1597: As a PX Blue developer, I want a maintainers/contributors page on Doc-it
* PXBLUE-1600: As Ovi, I want the PX Blue CLI to create an Angular 10 project
* PXBLUE-1601: As a PX Blue developer, I want to CLI to use all of the latest dependencies for React and React Native

File: src/docs/release-notes/R18/R18.mdx (hunk 4 above)
Comment (huayunh) [reply src/docs/release-notes/R18/R18.mdx:26]: And end the fight of `stickersheet` vs `sticker sheet`

**Comment by joebochill** [comment]
> Do we want to mention
> 
> * PXBLUE-1597: As a PX Blue developer, I want a maintainers/contributors page on Doc-it
> * PXBLUE-1600: As Ovi, I want the PX Blue CLI to create an Angular 10 project
> * PXBLUE-1601: As a PX Blue developer, I want to CLI to use all of the latest dependencies for React and React Native

I think we can leave these off they are kinda low level and covered under some of the bigger items I think ("improvements").

**Review by emclaug2 — APPROVED** [review]

**Pull request merged**